geopyspark-backend/geotrellis/src/main/scala/geopyspark/geotrellis/RasterRDD.scala

```diff
@@ -56,6 +57,32 @@ abstract class TileRDD[K: ClassTag] {
   def rdd: RDD[(K, MultibandTile)]
   def keyClass: Class[_] = classTag[K].runtimeClass
   def keyClassName: String = keyClass.getName
+
+  def reclassify(
+    intMap: java.util.Map[Int, Int]
+  ): TileRDD[_] = {
+    val scalaMap = intMap.asScala.toMap
+
+    reclassify(scalaMap.keySet, BreakMap(scalaMap))
+  }
+
+  def reclassifyDouble(
+    doubleMap: java.util.Map[Double, Double]
+  ): TileRDD[_] = {
```

Comment:
I think most of this got settled through in person discussion.

But to respond, and make sure we are on the same page:

1. doesn't seem necessary. And it seems like `BreakMap` makes more sense, and doens't allow 'sparce' reclass anyway.

2. Seems to me that following convention makes sense, but is it possible to allow the ability to override it with increased specificity? (perhaps this would look like default inclusive on the left, and exclusive on the right, but using a bracket instead of a paren overrides it? Maybe thats confusing.) Looks like Jake updated this already. Is it possible to make overriding the default convention an optional arg?)


(0:5 -> 1)
(5:10 ->2)